buildSrc/src/main/java/org/elasticsearch/gradle/testclusters/TestClustersPlugin.java

```diff
@@ -376,16 +368,16 @@ private static void configureCleanupHooks(Project project) {
         });
 
         project.getGradle().buildFinished(buildResult -> {
-            logger.info("Build finished");
+            logger.info("Build finished: {}", project.getPath());
```

Comment:
This seems pretty unnecessary. I'm certain there are already messages in the INFO log indicating the build has finished.